lib/puppet_x/puppetlabs/strings/yard/templates/default/template_helper.rb

```diff
@@ -73,9 +75,17 @@ def extract_param_details(parameters, tags_hash, fq_name = false)
       end
       if !param_exists
         parameter_info.push({:name => tag.name, :desc => tag.text, :types => tag.types, :exists? => false})
+        if location.length < 1 and location[0].length == 2
+          file_name = location[0][0]
+          line_number = location[0][1]
+          $stderr.puts "[warning] #{file_name}: #{line_number} The parameter #{tag.name} is documented, but doesn't actually exist in your code."
```

Comment:
what is the difference between "actually exist" and just "exist"? Take out the word "actually"